sdks/go/test/integration/io/fhirio/fhirio_test.go

```diff
@@ -75,13 +84,16 @@ func setupFhirStore(t *testing.T) (string, []string, func()) {
 	healthcareDataset := fmt.Sprintf(datasetPathFmt, *gcpopts.Project, *gcpopts.Region)
```

Comment:
Is this the standard way to define a dataset for Apache beam? Would we ever want to potentially have different datasets for different tests?

Reply:
I don't think it would ever be necessary to have a different dataset for different tests since we don't really care about the datasets, only the stores themselves matter here.
Did you have a particular way to define it in mind?